lib/vimeo/oauth.js

```diff
@@ -13,16 +13,29 @@ const fetch = require('isomorphic-unfetch')
 
 const {
   VIMEO_APP_KEY,
-  VIMEO_APP_SECRET
+  VIMEO_APP_SECRET,
+  VIMEO_APP_ACCESS_TOKEN
```

Comment:
IMO we could should drop `VIMEO_APP_KEY` and `VIMEO_APP_SECRET` since we need a access token with more permissions than a regular `client_credentials` would grant. And configuring both is just bloat. And not doing it will result in annoying warnings. Alt: just drop the warnings if `VIMEO_APP_ACCESS_TOKEN` is set.